### ISSUE TITLE
package-lock.json changes unexpectedly in npm-check-updates versions >= 16.4.0

### ISSUE THREAD

**Issue by jashan777** [raineorshine/npm-check-updates#1347]
- [✔️] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [✔️] I am using the latest version of `npm-check-updates`
- [✔️] I am using `node >= 14.14`

---

## Steps to Reproduce
node version - v14.20.0
npm version - v6.14.17
os - windows 11 or macos ventura 13.4
1. add npm-check-updates dependency in package.json file, version >= v16.4.0 
2. do `npm i` 
3. stage the changes, package.json and package-lock.json
4. remove the node_modules folder 
5. do `npm i` again.

package-lock.json will change 
these changes are reverted 

ScreenShots 
Below are the changes in package-lock.json the second `npm i`
![Screenshot 2023-11-04 120815](https://github.com/raineorshine/npm-check-updates/assets/49744054/de1d6e8b-9625-4384-a1e9-afc8fede445f)
![Screenshot 2023-11-04 120833](https://github.com/raineorshine/npm-check-updates/assets/49744054/7a777109-655e-41d6-a827-1ca36e0e0ab4)
![Screenshot 2023-11-04 120850](https://github.com/raineorshine/npm-check-updates/assets/49744054/530fc376-a482-4484-b250-4a7eac57ff69)
![Screenshot 2023-11-04 120951](https://github.com/raineorshine/npm-check-updates/assets/49744054/4f1d07b3-27ae-4515-ad40-fc55a6d6656f)
![Screenshot 2023-11-04 121008](https://github.com/raineorshine/npm-check-updates/assets/49744054/e2ab2e72-d195-4e07-a258-95f265e6b72b)

**Comment by jashan777** [comment]
seems to be a node/npm version issue as the issue doesn't appear in node version >=v16.19.0 and  npm >=8.19.3

**Comment by raineorshine** [comment]
Yeah, this is more an npm issue than npm-check-updates. We don't touch the package lock file, and we don't have control over how the modules are installed.

**Comment by jashan777** [comment]
> Yeah, this is more an npm issue than npm-check-updates. We don't touch the package lock file, and we don't have control over how the modules are installed.

The changes are reverted if one remove the node_modules folder again and do npm i , but supprisingly npm-check-updates version <= 16.3.0 works just fine and doesnt cause this issue.

**Comment by raineorshine** [comment]
Yes, that does seem strange. I wonder what changed between 16.3.0 and 16.4.0 that changed npm's install behavior.
